BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -161,6 +162,9 @@ final class LocationView: UIView, LocationTextFieldDelegate, ThemeApplicable, Ac
         iconContainerStackViewLeadingConstraint?.isActive = true
 
         NSLayoutConstraint.activate([
+            dropDownSearchEngineView.heightAnchor.constraint(equalToConstant: UX.dropDownImageViewSize.height),
+            dropDownSearchEngineView.widthAnchor.constraint(equalToConstant: UX.dropDownImageViewSize.width),
+
```

Comment:
Can we instead move all the drop down button's sizing-related constraints into `DropDownSearchEngineView`?

```
UX struct:
static let searchEngineImageSize = CGSize(width: 24, height: 24)
static let chevronImageSize = CGSize(width: 8, height: 8)
```

```
Constraints:
searchEngineImageView.widthAnchor.constraint(equalToConstant: UX.searchEngineImageSize.width),
searchEngineImageView.heightAnchor.constraint(equalToConstant: UX.searchEngineImageSize.height),
...
arrowImageView.widthAnchor.constraint(equalToConstant: UX.chevronImageSize.width),
arrowImageView.heightAnchor.constraint(equalToConstant: UX.chevronImageSize.height),
```
That way, the `DropDownSearchEngineView` will be able to fully define its own size, and LocationView doesn't need to know any specifics for either `SearchEngineView`. 😃  (Also, total width will be `42` once all the margins are considered, not `40`)